apps/studio/components/interfaces/Storage/AnalyticsBuckets/AnalyticsBucketDetails/SimpleConfigurationDetails.tsx

```diff
@@ -37,7 +37,6 @@ export const SimpleConfigurationDetails = ({ bucketName }: { bucketName?: string
             >
               Learn more
             </InlineLink>
-            .
           </ScaffoldSectionDescription>
```

Comment:
Unrelated but I'm wondering whether this inline link might be better as a button beside "Copy all"

Reply:
Opted against this because there is already a _Docs_ button up top. Felt like two button-level docs-like buttons would get confusing.

Or did you have another idea for what this button would say, aside from _Docs_?

Reply:
oh thats true - yeah that plus there's starting to be a lot of buttons as well
- Docs
- View replication
- Add tables
- (If schema not synced) Connect a table

agree lets just keep it inline hahah